Firestore/Example/Tests/Integration/API/FIRIndexingTests.mm

```diff
@@ -29,15 +33,15 @@ @implementation FIRIndexingTests
 - (void)setUp {
   [super setUp];
   self.db = [self firestore];
-  XCTestExpectation* exp = [self expectationWithDescription:@"clear persistence"];
-  [self.db clearPersistenceWithCompletion:^(NSError*) {
+  XCTestExpectation *exp = [self expectationWithDescription:@"clear persistence"];
```

Comment:
why the reposition of *? Is this automatic?

Reply:
I try to revert it, but every time I run formatter it comes back